### PR TITLE
📄 Feature EPL-819: Documentation about cssCompile task

### PR DESCRIPTION
**Pull request by tomyrodeghiero**
**Screenshots** of the documentation developed on the **Gradlew** task of `cssCompile` are attached below.

image
image
image
image
image

### CODE REVIEW

**Review by juabaez — APPROVED** [review]

**Review by Gremiger — APPROVED** [review]

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+## Overview
+
+The `cssCompile` task in the **Etendo** Gradle configuration is specifically designed to convert `.scss` files into `.css` files. This documentation aims to provide **Etendo partners** with a comprehensive understanding of its function, usage, and benefits.
+
+## Purpose
+
+With modern web development, **SCSS** offers more power and flexibility compared to traditional **CSS**. As Etendo aims to deliver an impeccable user interface, using `.scss` files allows for better organization, variable definitions, and nested rules. However, browsers understand only CSS. Hence, there's a need to **compile SCSS into CSS**, and that's where our `cssCompile` task steps in.
+
+> **Requirements**
+> - npm (Node Package Manager) installed and Sass compiler installed on your system.
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:14]: Missing block style

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+!!! info
+    ### Installing npm (Node Package Manager)
+    If you don't have npm installed on your system, follow these steps:
+    
+    - Install npm globally using the following command:
+    
+    ```bash
+    npm install -g npm
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:23]: On mac os and windows it is installed with the same commands?
If they are different commands, you can make a tab for each OS.
E.g:
image

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    - Confirm the Sass installation by running:
+
+    ```bash
+    sass --version
+    ```
+    
+    Seeing the Sass version number means that Sass has been installed correctly.
+
+
+## How It Works
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:51]: I don't think it is necessary to explain how the task works.

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+2. Navigate to the root directory of the **Etendo project**.
+3. Run the following **command**:
+
+```bash
+./gradlew cssCompile smartbuild --info
+```
+
+!!! success
+    After execution, you should see an output similar to:
+    > Task :cssCompile
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:100]: Missing block style

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+## Troubleshooting
+
+If you encounter the error `A problem occurred starting process 'command 'sass''`, it suggests that the `sass` compiler either isn't installed or isn't accessible. Follow these checks:
+
+1. Ensure you've installed the **Sass** compiler.
+2. Verify that the `sass` command is available in your system's **PATH**. You can do this by running the `sass --version` command.
+3. If the problem persists, **restart your terminal** or ensure you have the necessary permissions to execute commands.
+
+
+## Conclusion
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:116]: I think this conclusion is not necessary, instead I would add an example by changing the size of the navigation bar buttons and compiling again. 
@lucas-palacios what do you think?

**Review by isaiasb-etendo — CHANGES_REQUESTED** [review]

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md (hunk 1 above)
Comment (tomyrodeghiero) [reply docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:14]: Perfect, thank you for the comment!

File: docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md (hunk 2 above)
Comment (tomyrodeghiero) [reply docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md:23]: Yes Isa, in this case the provided commands are consistent across all platforms (Windows, macOS, and Linux).

**Pull request closed without merging**